### ISSUE TITLE
Training on coco2017

### ISSUE THREAD

**Issue by zhwuer** [matterport/Mask_RCNN#220]
Hi 
I want to train this mask rcnn on coco2017, but there are no valminusminival nor minival

**Comment by zhwuer** [comment]
I just find one solution:
move this part of code 
>         if subset == "minival" or subset == "valminusminival":
               subset = "val"
above
> coco = COCO("{}/annotations/instances_{}{}.json".format(dataset_dir, subset, year))
but I don't know whether it is right.

**Comment by zhwuer** [comment]
The solution is just delete anything you don't need in using coco2017.
I am so stupid.

**Comment by ccnankai** [comment]
Hi   @zhwuer   ，
I encountered the same problem. How did you solve it?

**Comment by mathpopo** [comment]
# Training dataset. Use the training set and 35K from the
    # validation set, as as in the Mask RCNN paper.
    dataset_train = CocoDataset()
    dataset_train.load_coco(args.dataset, "train", year=args.year, auto_download=args.download)
    dataset_train.prepare()

    # Validation dataset
    dataset_val = CocoDataset()
    dataset_val.load_coco(args.dataset, "val", year=args.year, auto_download=args.download)
    dataset_val.prepare()
